packages/gatsby-transformer-sharp/src/create-resolvers.js

```diff
@@ -0,0 +1,18 @@
+module.exports = ({ createResolvers, reporter }) => {
+  const resolvers = {
+    File: {
+      childImageSharp: {
+        resolve: (parent, args, context, info) => {
+          if (parent.extension === `gif`) {
+            reporter.warn(
```

Comment:
This has potential of spamming console. Do we need this warning for each instance of it?

Reply:
@pieh I guess we could also collect the path and print out only one warning with all paths at the end, e.g. with `onPostBootstrap` or something?

Reply:
Printing it for every instance might be fine. I just wanted to raise potential problems it can cause.

I wouldn't recommend using `onPostBootstrap` (as this will result in non-warning cases for gatsby develop (local / preview) as people change their data or queries in source code).

Ideally we would have "sessions" for each "query running" step where we dedupe warnings for same files within those, but I don't think plugins can do something like that right now.

I'll try to find "prior art" in other plugins for similar warnings and if we have any - let's follow what we already do in other plugins

Reply:
Let's keep it like that then for now

Reply:
@pieh shortened the message now